agreement.md

```diff
@@ -0,0 +1,151 @@
+# Aragon Network Agreement
+
+## Section 0. Background
+The Aragon Network is a Decentralized Autonomous Organization that builds and maintains a dispute resolution protocol and other smart contract infrastructure.
+
+The Aragon Network is governed by Aragon Network Token (ANT) holders, who supported the initial creation of the network and under this Agreement will continue to guide its evolution through established governance processes.
+
+This Agreement exists to enumerate the shared mission, principles, and governance guidelines that bind ANT holders in their collaborative effort to create value for the Aragon Network.
+
+## Section 1. RFC 2119 compliance
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+## Section 2. Immutable guidelines
```

Comment:
Idea: A common discussion/debate around constitutional law is over how to interpret a constitution. There are "loose constructionists", "strict constructionists", "textualists", "originalists", etc. Should we create an immutable guideline saying explicitly how the Agreement should be interpreted, to remove this debate altogether (or at least to shift the debate to a more productive topic)?